docs/developer-guide/release.md

```diff
@@ -50,13 +50,13 @@ _Note: While the release is running, don't shut your laptop or switch between re
 1. Do a docker pull: `docker pull obolibrary/odkfull:dev`
```

Comment:
No longer need to run this line: Do a docker pull: `docker pull obolibrary/odkfull:dev`